instructions.json

```diff
@@ -18,25 +18,35 @@
             "defaultInstruction": "Head {modifier}[ on {way_name}]"
         },
         "end of road": {
-            "defaultInstruction": "Turn {modifier}[ onto {way_name}]"
+            "defaultInstruction": "Turn {modifier}[ onto {way_name}] at the end of the road",
```

Comment:
Well, if its only about minimising the wording in a restricted setting, then using `turn` is probably the safer bet. I have so say though, that I fear we might limit ourselves here if this repo is going to be the main point to get instructions from. I might have a conflicting impression of what we want to achieve here, though.